README.md

```diff
@@ -1,4 +1,10 @@
 ﻿# Revisions
```

Comment:
What do you think about moving Revisions after the intro?

Reply:
Maybe once things are a little further along in development. Right now since its in heavy development I feel like revisions are probably more important.